### PR TITLE
tag v1.0.3

### PR DESCRIPTION
**Pull request by kokeshiM0chi**
## Description
For tagging v1.0.3

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+### BREAKING CHANGE
+- Nothing
+
+### FEATURES
+- Nothing
+
+### IMPROVEMENTS
+- [p2p/peer] [\#341](https://github.com/line/ostracon/pull/341) Remove default case
+- [review] [\#346](https://github.com/line/ostracon/pull/346) Add CODEOWNERS
+- [upgrade] [\#349](https://github.com/line/ostracon/pull/349) Main patch from tm-v0.34.9
```
Comment (tnasu) [inline CHANGELOG.md:17]: I prefer to use `backport/tendermint` or `bp/tendermint`. How about other CODEOWNERS?

**Review by tnasu — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## v1.0.3
+
+*Jan 20, 2022*
```
[the remark this answers is not in the document]
Comment (kokeshiM0chi) [reply CHANGELOG.md:5]: Thank you. I will fix it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+
+### BREAKING CHANGE
+- Nothing
+
+### FEATURES
+- Nothing
+
+### IMPROVEMENTS
+- [p2p/peer] [\#341](https://github.com/line/ostracon/pull/341) Remove default case
+- [review] [\#346](https://github.com/line/ostracon/pull/346) Add CODEOWNERS
```
[the remark this answers is not in the document]
Comment (kokeshiM0chi) [reply CHANGELOG.md:16]: Thank you. I will fix it.

File: CHANGELOG.md (hunk 1 above)
Comment (kokeshiM0chi) [reply CHANGELOG.md:17]: I think `backport/tendermint` is better, too.

**Review by torao — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+## v1.0.3
+
+*Jan 20, 2022*
+
+* Improve p2p/peer reactor
```
Comment (tnasu) [inline CHANGELOG.md:7]: Could you add more information about backports?

**Review by tnasu — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (kokeshiM0chi) [reply CHANGELOG.md:7]: OK. I will fix it!

**Review by torao — APPROVED** [review]

**Review by tnasu — APPROVED** [review]

**Review by torao — APPROVED** [review]

**Pull request merged**